.env.example

```diff
@@ -23,6 +23,7 @@ USE_EXTERNAL_AUTHENTICATION=false
 EXTERNAL_AUTHENTICATION_USER_HEADER=""
 JWT_ALGORITHM=
 JWT_SECRET=
+JWT_AUDIENCE=
```

Comment:
Can we call it JWT_CLIENT_ID instead to make it clearer what it is?

Reply:
We could. I had named it audience since the Python piece was looking for audience and it was clear there. I'm open either way.